project-codeflare/codeflare-sdk
# fix: get_cluster logic

# Issue link


References [RHOAIENG-4028](https://issues.redhat.com/browse/RHOAIENG-4028)
# What changes have been made

The addition of the new volumeMounts has caused the `get_cluster()` function to fail as it assumes the new VMs are the local interactive VMs. 
This makes the function try to recreate the local_interactive related resources which causes the 
```ValueError: ingress_domain is invalid. For creating the client route/ingress please specify an ingress domain``` 
error because `ingress_domain` would not be specified.

Also added simplified import for the `get_cluster()` command. It can now be imported like this.
`from codeflare_sdk import get_cluster`
# Verification steps



Run through a demo notebook and create a Ray Cluster on Kind and OpenShift.
run `cluster = get_cluster(name, namespace)` 
`cluster.down()` and other `cluster.` commands should work.
## Checks
- Testing Strategy
